### PR TITLE
Nader js 3 week 2

### PR DESCRIPTION
**Pull request by naderakhgari**
# Your Details

Your Name:
Your City:
Your Slack Name:

# Homework Details

Module:
Week:

### CODE REVIEW

File: week-8/Homework/mandatory/1-practice/1-practice.md [hunk 1]
```diff
@@ -20,12 +20,19 @@ Enjoy!
 
 The following endpoint is publicly available from Github
 
-    GET https://api.github.com/repos/{owner}/{repo}/pulls/{pull_number}/comments
+    GET https://api.github.com/repos/CodeYourFuture/js-exercise/pulls/688/comments
 
 1. Describe in a sentence what this API endpoint returns.
 
 <!-- Write your answer here -->
+<!--This API return an array of objects from pull_number of repo of owner-->
```
Comment (textbook) [inline week-8/Homework/mandatory/1-practice/1-practice.md:28]: Note that when you put these in comments, they're not visible in the rendered version: https://github.com/naderakhgari/js-exercises/blob/nader-js-3-week-2/week-8/Homework/mandatory/1-practice/1-practice.md

File: week-8/Homework/mandatory/3-dog-photo-gallery/script.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        console.log(data);
+        let listShow = document.getElementById("listShow");
+        let listEle = document.createElement("li");
+        let img = document.createElement("img");
+        listShow.appendChild(listEle);
+        listEle.appendChild(img);
+        img.src = data.message;
+    })
+    .catch(error =>{
+        console.log(error);
```
Comment (textbook) [inline week-8/Homework/mandatory/3-dog-photo-gallery/script.js:18]: Note that this is only visible to developers, most people don't look at the console - when handling errors, think if there's any information the end user would need to understand what happened

File: week-8/Homework/mandatory/1-practice/1-practice.md (hunk 1 above)
Comment (naderakhgari) [reply week-8/Homework/mandatory/1-practice/1-practice.md:28]: thanks for your notice. I got it.

File: week-8/Homework/mandatory/3-dog-photo-gallery/script.js (hunk 2 above)
Comment (naderakhgari) [reply week-8/Homework/mandatory/3-dog-photo-gallery/script.js:18]: I think it's good to use of 'console.error(error)'

**Pull request closed without merging**